### ISSUE TITLE
Update bottom part of README

### ISSUE THREAD

**Issue by fool** [netlify/prerender#19]
still mentions wrong things about the old PhantomJS implementation

**Comment by fool** [comment]
not sure if y'all want to take on docs for these kinds of repos, @klavavej , so feel free to remove from your project if not.

**Comment by klavavej** [comment]
@fool - we would consider issues for these kinds of repos if they were high priority. Since this is low priority I'm going to remove it from our board.

**Comment by fool** [comment]
Yup, hard to consider this one high priority.  Guess I'll do it myself,
someday, well that means it will never happen so I'll leave the issue there
for whomever touches the repo.  Thanks for being frank @klavavej

**Comment by fool** [comment]
While this is definitely long in the tooth, it is still needed - those docs are flat out wrong, so I've opted this out of the stalebot workflow.

**Comment by fool** [comment]
oh, this is kinda a dupe of https://github.com/netlify/prerender/issues/29
